samples/tracebld/trcbld.cpp

```diff
@@ -4266,7 +4266,7 @@ PWCHAR LoadCommandLine(PCWSTR pwz, PWCHAR pwzDst, PWCHAR pwzDstEnd)
             // More arguments!
             WCHAR wzPath[MAX_PATH];
             PWCHAR pwzPath = wzPath;
-            PCWSTR pwzTmp = pwzArgBeg + 1;
+            PCWSTR pwzTmp = pwzArgBeg;
```

Comment:
This is a fix for an off-by-one error in response file parsing. Response file parsing doesn't work at all without this fix.